packages/discord.js/src/managers/GuildScheduledEventManager.js

```diff
@@ -276,8 +277,26 @@ class GuildScheduledEventManager extends CachedManager {
 
     let { limit, withMember, before, after } = options;
 
-    const data = await this.client.api.guilds(this.guild.id, 'scheduled-events', guildScheduledEventId).users.get({
-      query: { limit, with_member: withMember, before, after },
+    const query = new URLSearchParams();
+
+    if (limit) {
+      query.set('limit', limit);
+    }
+
+    if (withMember) {
```

Comment:
Won't set the query if false

Reply:
lol, got lucky that discord's default is false 💀, will fix